libs/features/components/ui-lists/src/lib/elewa-group-horizontal-list-orgs/elewa-group-horizontal-list-orgs.component.html

```diff
@@ -0,0 +1,5 @@
+<div class="horizontal-list-container">
+    <div class="partner" *ngFor="let partner of partners">
+      <img [src]="partner.logoUrl" alt="{{ partner.name }}">
+    </div>
+</div>
```

Comment:
Please provide a mobile view screenshot for your work, the logos need to be in pairs of 2 in each row on mobile view.